provisioning/stream/local.sh

```diff
@@ -56,6 +56,9 @@ export STREAM_VOLUME_FAST_STORAGE_SIZE="100Mi"
 export STREAM_VOLUME_MEDIUM_STORAGE_SIZE="100Mi"
 export STREAM_VOLUME_SLOW_STORAGE_SIZE="100Mi"
 
+export STREAM_ENCRYPTION_PROVIDER="native"
+export STREAM_ENCRYPTION_NATIVE_SECRET_KEY="00000000000000000000000000000000"
```

Comment:
Where to take the secret key for localhost from? I could randomly generate it but that doesn't feel technically correct since it should be preserved between deploys. :thinking:

Reply:
I would expect similarly to the library. It has some local provisioning. As it is shell script you can use any random number generator basically. E.g get random number from `/dev/random` https://unix.stackexchange.com/questions/268952/using-dev-random-dev-urandom-to-generate-random-data

Reply:
In case of the library you just need to run tests so it doesn't matter if the generated secret key ends up being different every time. Here however you might want to locally deploy one version, then change something and locally deploy the the fix in which case the key should stay the same.